src/main/java/com/checkout/payments/PaymentsQueryFilter.java

```diff
@@ -0,0 +1,19 @@
+package com.checkout.payments;
+
+import javax.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public final class PaymentsQueryFilter {
```

Comment:
Add `NoArgsConstructor` please, this is just to make more friendly the constructor.